valio-master/Valio/ScheduleTableViewController.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  ScheduleTableViewController.swift
+//  Valio
+//
+//  Created by Sam Soffes on 6/5/14.
+//  Copyright (c) 2014 Sam Soffes. All rights reserved.
+//
+
+import UIKit
+
+class ScheduleTableViewController: UITableViewController {
+
+	lazy var 📅: NSArray = {
```

Comment:
For a fun project it's ok, but I wouldn't use emoji variable names in production, because it's visually distracting, probably can cause problems in other tools parsing the source code, etc.